### PR TITLE
Fluff Stuff

### PR DESCRIPTION
**Pull request by stobarico**
Adds/Replaces: https://i.imgur.com/egB0KUW.png

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -159,6 +159,53 @@
...
 	item_color = "centcom"
 
+/obj/item/clothing/under/solgov
+	name = "\improper Sol Federation marine uniform"
+	desc = "A comfortable and durable combat uniform worn by Sol Federation Marine Forces."
+	icon_state = "solgov"
+	item_state = "ro_suit"
+	item_color = "solgov"
+	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	displays_id = 0
```
Comment (stobarico) [inline code/modules/clothing/under/miscellaneous.dm]: **can_adjust = 0**

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -159,6 +159,53 @@
...
+	item_color = "solgov"
+	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	can_adjust = 0
+
+/obj/item/clothing/under/solgov/command
+	name = "\improper Sol Federation Lieutenant uniform"
+	desc = "A comfortable and durable combat uniform worn by Sol Federation Marine Forces. This one has additional insignia on its shoulders."
+	icon_state = "solgovc"
+	item_color = "solgovc"
+	can_adjust = 0
```
Comment (AuroranAI) [inline code/modules/clothing/under/miscellaneous.dm]: can_adjust is inherited

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -322,6 +322,18 @@
...
 	breakouttime = 3000
 
+/obj/item/clothing/suit/nt_greatcoat
+	name = "\improper Nanotrasen Greatcoat"
+	desc = "A greatcoat with Nanotrasen colors."
+	icon_state = "nt_jacket"
+	item_state = "nt_jacket"
+	blood_overlay_type = "coat"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	allowed = list(/obj/item/weapon/tank/internals/emergency_oxygen,/obj/item/weapon/reagent_containers/spray/pepper,/obj/item/device/flashlight,/obj/item/weapon/gun/energy,/obj/item/weapon/gun/ballistic,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/storage/fancy/cigarettes,/obj/item/weapon/lighter,/obj/item/device/detective_scanner,/obj/item/device/taperecorder,/obj/item/weapon/melee/classic_baton)
```
Comment (AuroranAI) [inline code/modules/clothing/suits/miscellaneous.dm]: spaces between items

File: ftl13.dme [hunk 4]
```diff
@@ -1,2239 +1,2239 @@
...
-#include "_maps\__MAP_DEFINES.dm"
-#include "_maps\basemap.dm"
-#include "code\_compile_options.dm"
-#include "code\world.dm"
-#include "code\__DATASTRUCTURES\globals.dm"
-#include "code\__DATASTRUCTURES\heap.dm"
-#include "code\__DATASTRUCTURES\linked_lists.dm"
-#include "code\__DATASTRUCTURES\priority_queue.dm"
-#include "code\__DATASTRUCTURES\stacks.dm"
-#include "code\__DEFINES\access.dm"
```
Comment (AuroranAI) [inline ftl13.dme]: Redo your .dme changes,  it shouldn't be changing every single line

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (AuroranAI) [inline code/modules/clothing/under/miscellaneous.dm]: all of these can_adjust should be FALSE instead of 0

**Pull request closed without merging**